montblanc/impl/biro/v2/loaders/loaders.py

```diff
@@ -23,6 +23,10 @@
 import montblanc
 import montblanc.impl.common.loaders
 
+ORDER_CASA = 'casa'
+ORDER_OTHER = 'other'
+VALID_ORDERINGS = [ORDER_CASA, ORDER_OTHER]
```

Comment:
I think the use of ORDER_CASA, ORDER_OTHER is redundant.
I would suggest to write directly:
       if uvw_order == 'casa':
               ...
        elif uvw_order == 'montblanc':
               ...
        else: